packages/snaps-utils/src/namespace.ts

```diff
@@ -25,6 +25,8 @@ export const CHAIN_ID_REGEX =
 export const ACCOUNT_ID_REGEX =
   /^(?<chainId>(?<namespace>[-a-z0-9]{3,8}):(?<reference>[-a-zA-Z0-9]{1,32})):(?<accountAddress>[a-zA-Z0-9]{1,64})$/u;
 
+export const ACCOUNT_ADDRESS_REGEX = /^(?<accountAddress>[a-zA-Z0-9]{1,64})$/u;
```

Comment:
What is this regex supposed to be?

Reply:
Account address regex used in creating the `AccountAddress` which is used in the `OnNameLookupResponse` type.